Packs/ApiModules/Scripts/TAXII2ApiModule/TAXII2ApiModule.py

```diff
@@ -240,6 +241,8 @@ def init_roots(self):
             self.api_root = self.server.api_roots[0]  # type: ignore[union-attr, attr-defined]
             # override _conn - api_root isn't initialized with the right _conn
             self.api_root._conn = self._conn  # type: ignore[attr-defined]
+        # enable logging
+        logging.disable(logging.NOTSET)
```

Comment:
lets put that in `finally` clause of the `try-except` so it will be disable even if theres a failure